server/container/guice/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/JMAPModule.java

```diff
@@ -105,6 +106,10 @@ protected void configure() {
         bind(MailQueueItemDecoratorFactory.class).to(PostDequeueDecoratorFactory.class).in(Scopes.SINGLETON);
 
         Multibinder.newSetBinder(binder(), MailboxListener.GroupMailboxListener.class).addBinding().to(PropagateLookupRightListener.class);
+
+        Multibinder<Version> supportedVersions = Multibinder.newSetBinder(binder(), Version.class);
+        supportedVersions.addBinding().toInstance(Version.DRAFT);
+        supportedVersions.addBinding().toInstance(Version.RFC8621);
```

Comment:
why it's in jmap-draft module?

Reply:
That's what I thought too before getting this: https://github.com/linagora/james-project/pull/3211#discussion_r403850224

I would tend to agree about the fact that it shouldn't be in the draft module. I think we should add the RFC version into a separate new jmap module protocol I guess? But I don't think this work is for this PR

Reply:
I dropped it at the end